apps/sv/frontend/src/__tests__/sv.test.tsx

```diff
@@ -53,6 +53,66 @@ describe('SV user can', () => {
 
     expect(await screen.findByText('Vote Requests')).toBeDefined();
   });
+
+  test('see proper time format in popup', { timeout: 10000 }, async () => {
+    const mockedDate = new Date(2020, 0, 14, 4, 42, 0);
+
+    vi.setSystemTime(mockedDate);
```

Comment:
Nice, first time seeing this api. I was about to ask how 2020 works until i saw this.